### DISCUSSION TITLE
Slightly rename menu items?

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#79] [Q&A] [answered]
I am considering to rename some of the menu items to make the naming more consistent with [existing books and articles](https://github.com/probonopd/MiniDexed/wiki/Recommended-reading). @rsta2 I think I could make the needed changes in the source code but I don't know whether it'd have any unintended "dangerous" side effects.

Would it?

```
No Effect
Total volume
Output balance(A/B)
PAN control
Dual detune
Algorithm
Feedback level
LFO Wave
LFO Speed
LFO Delay time
LFO P MOD. SENS.
LFO P MOD. Depth
LFO A MOD. Depth
Pitch EG Rate 1
Pitch EG Rate 2
Pitch EG Rate 3
Pitch EG Rate 4
Pitch EG Level 1
Pitch EG Level 2
Pitch EG Level 3
Pitch EG Level 4
Portamento time
Frequency coarse
Frequency fine
OSC. detune
EG Rate 1 (R1)
EG Rate 2 (R2)
EG Rate 3 (R3)
EG Rate 4 (R4)
EG Level 1 (L1)
EG Level 2 (L2)
EG Level 3 (L3)
EG Level 4 (L4)
Key Velocity
AMP. MOD. SENS.
Total level
```

**Comment by rsta2** [comment, marked as answer]
Not dangerous, but the LCD display may show nonsense. We have only 12 characters for a parameter name, because we need 4 characters for the " TG#" or " OP#" in the right corner. I had already kept this in mind, when implementing the `CUIMenu` class. I guess there is not much room for improvement here.

**Reply by probonopd** [reply]
Ah, right. Not that I think the original names are super intuitive or consistent either ;-)

**Reply by rsta2** [reply]
No, but they obviously had the same problem (2x16 characters). ;)
